### ISSUE TITLE
Local front-end development connected to the local backend stack?

### ISSUE THREAD

**Issue by dionjwa** [nhost/nhost#1066]
**tl;dr** the documentation was elsewhere, noted in the next comment

I am going through the first steps, and all is well but:
1. I have a local backend stack running (`nhost up`) 👍🏻 
1. The front-end requires: ```REACT_APP_NHOST_SUBDOMAIN="XXX"
REACT_APP_NHOST_REGION="us-east-1"```
2. So it points to the remote backend

So I cannot develop the local frontend against the local backend stack? 

```mermaid
graph LR
  local-stack
  b[local browser] --> |auth OK| remote-stack
  local-stack --> |deploys OK| remote-stack
  b --> |auth and graphql mutations possible?| local-stack
  		
```

To be clear:
 - local hasura migrations+config to remote is good
 - browser auth is good
 - but I cannot mutate state via the local frontend because the remote doesn't yet have the needed migrations I'm working locally on, and I'm not actually connected to the remote backend with the local frontend? Maybe I'm missing something, but I would understand this might be complicated with auth+email etc.

**Comment by dionjwa** [comment]
Ah! The information IS in the docs, but was in the CLI section, not the getting started section:

https://docs.nhost.io/platform/cli#subdomain-and-region

You can close this, it works as expected, but maybe here a note for the docs layout. The information was there, but not in the expected place.

Thank you! Good experience so far, good job.

**Comment by elitan** [comment]
Thanks for letting us know. I made some small updates to our docs here: https://github.com/nhost/nhost/pull/1068

I added the following: 

![CleanShot 2022-10-24 at 08 26 38](https://user-images.githubusercontent.com/331818/197461278-a0246509-ffd5-4fbc-977c-b14142fcef61.png)


What do you think?

**Comment by elitan** [comment]
Closing the issue officially since the question has been answered.
